__tests__/api.test.js

```diff
@@ -27,3 +28,14 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api", () => {
+  test("status 200 - JSON describing all the available endpoints on my API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        expect(JSON.parse(res.body.endpoint)).toEqual(endpoints);
```

Comment:
This is much better!
Points for improvement: 
You could iterate through the returned array and be more specific about what form the JSON object should take.

Maybe have a look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)